### DISCUSSION TITLE
SearXNG down on search.drivet.xyz

### DISCUSSION THREAD

**Discussion by ghost** [searxng/searx-instances#203] [Report instance downtime]
Unfortunately my whole server is down (fuck Ovhcloud) which took all my services down. Nothing i can do at the moment to fix this so yeah thank ovh and dont blame me

**Comment by unixfox** [comment]
Thank you for notifying us about it. As we thrive to get an instance list with instances that are working we temporarily removed your instance in https://github.com/searxng/searx-instances/commit/5832c892e342c3fb4416dfddc29bfd7e8b9461f3

But you can request to add it back in https://github.com/searxng/searx-instances/issues/200

**Reply by ghost** [reply]
Thank you for fast response. I will request to get the instance back to the list when my server comes back up
